material-kit-pro-react-v1.9.0/src/Testing.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
```

Comment:
@HassenH1 why did we put Testing.js in material-kit-pro....directory.  I would prefer we put it in the a src/testing directory.